target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Device Tree file for the HooToo TM-05
+ *
+ * Copyright (C) 2020 Russell Morris (rmorris@rkmorris.us)
+ * 
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+/dts-v1/;
+
+#include "mt7620n.dtsi"
```

Comment:
@arrmo , is it worth to use a common `dtsi` with Ravpower WD03, did you checked?

Reply:
I think what you mean is (correct me if I'm wrong!) ... if the value is defined (the same) in mt7620n.dtsi, then remove it here ... right? No issue at all, if that's the way to go.

Reply:
Sorry, my bad!
I just asking here, as the HooToo-Ravpower common `dtsi` would have place here!

My DTS-fu is almost zero, so I don't know the answer to your question.

Reply:
You'll have to tell me offline if DTS-"fu" means the same there as here ... LMAO! But whatever it is, I think you know it better than I do 😄.

I'm all for removing items defined in a common file, just let me know. Thanks!

Reply:
It means that I don't know DTS. :upside_down_face:

Reply:
This was marked as resolved earlier, by @adrianschmutzler - so I think it's OK as it is, but yell if not!